src/packages/core/src/options/server-options.ts

```diff
@@ -49,5 +60,20 @@ export const ServerOptions: Definitions<ServerConfig> = {
       "Whether or not websockets should response with binary data (ArrayBuffers) or strings.",
     default: () => "auto",
     cliChoices: [true, false, "auto"] as any[]
+  },
+  rpcEndpoint: {
+    normalize,
+    cliDescription:
+      "Defines the endpoint route the HTTP and WebSocket servers will listen on.",
+    default: config => {
+      switch (config.flavor) {
+        case FilecoinFlavorName:
+          return "/rpc/v0";
+        case DefaultFlavor:
+        default:
+          return "/";
+      }
+    },
+    defaultDescription: "'/rpc/v0' for Filecoin, '/' otherwise"
```

Comment:
```suggestion
    defaultDescription: '"/" (Ethereum), "/rpc/v0" (Filecoin)'
```

Reason: I like to put ethereum things first. Also, shortening it because tezos's endpoint will be added in the future.